airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -287,7 +287,7 @@ protected void assertCdcSchemaQueryable(final JsonNode config, final JdbcDatabas
   protected void assertSqlServerAgentRunning(final JdbcDatabase database) throws SQLException {
     try {
       final List<JsonNode> queryResponse = database.unsafeQuery(connection -> {
-        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%'";
+        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%' OR  [servicename] LIKE 'SQL Server 代理%' ";
```

Comment:
Nitpick. There is an extra space after the `OR`.

```suggestion
        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%' OR [servicename] LIKE 'SQL Server 代理%' ";
```